docs/api.md

```diff
@@ -168,6 +167,112 @@ player.playlist.contains({
 // 4
 ```
 
+#### `player.playlist.indexOfLast() -> Number`
```

Comment:
🤔  I think I would prefer these methods to be `lastIndex`, `previousIndex`, `nextIndex`.
Thoughts?

Reply:
oh, just saw that you talked about this with @BrandonOCasey.
I'm not sure that consistency with `indexOf` is really something to strive for. `indexOf` was used to mirror the array api.
Looking to see if there are precedents for this.

Reply:
I'm cool with changing this - I don't feel strongly about the naming.